lib/ansible/galaxy/dependency_resolution/gpg.py

```diff
@@ -0,0 +1,298 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""Signature verification helpers."""
```

Comment:
Should this module live in the dep resolver code? It feels like it could be a separate isolated thing.
I see it's only imported from within `ansible.galaxy.collection` — why not have it in that subpackage?

Reply:
Only one function is used by dep resolver code, so keeping it in ansible.galaxy.collection makes more sense. I moved it.